backend/routes/api/parks.js

```diff
@@ -104,9 +105,17 @@ router.get(
       throw error;
     }
 
-    const output = park.toJSON();
+    const parkJson = park.toJSON();
 
-    res.json(output);
+    const subAreas = _.mapValues(
+      // group seasons by feature type
+      _.groupBy(parkJson.seasons, (s) => s.featureType.name),
+
+      // sort by year
+      (group) => _.orderBy(group, ["operatingYear"], ["desc"]),
+    );
+
+    res.json({ ...parkJson, subAreas });
```

Comment:
Temporary change in here to format the data differently. If this looks okay, we can make it permanent

Reply:
If we're returning the seasons grouped by their featureType. Then, we probably can get rid of the regular list of seasons to reduce the payload.

Reply:
Just pushed an update to delete "seasons" from the structure ✅